kite/__init__.py

```diff
@@ -1 +1 @@
-from kite.program import *
+from kite.program import *
```

Comment:
Need a newline

Reply:
yep, always forgetting that, newline is added now.